.circleci/config.yml

```diff
@@ -0,0 +1,278 @@
+version: 2
+
+jobs:
+  python-3.5-client-3:
+    docker:
+      - image: python:3.5
+      - image: redis:3
+    steps:
+      - checkout
+      - run: python -m pip install -r dev_requirements.txt
```

Comment:
Actually `dev_reuirements.txt` is only used for development, may be we should add a `requirements.txt` to distinguish?

Reply:
Sure thing, one sec